package.json

```diff
@@ -4,11 +4,13 @@
   "description": "Make travel and accommodation easy and convenient.",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node src/index.js",
+    "build": "babel src --out-dir dist",
+    "start": "node dist/index.js",
```

Comment:
Looks like your start script will not start the app. Have your:
- `build` script create the `dist` folder
- `start` script run the build script then also start the app